master/usage/migrating-pools.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Changing IP pools
+---
+
+## About changing IP pools
+
+When using {{ site.prodname }} IPAM, each workload is assigned an address from the selection of configured IP pools.
+There are a few reasons you may want to change the IP pool being used on a running cluster, including moving to a
+larger CIDR which can accomodate more workloads, or moving off of a CIDR that was used accidentally.
+
+### Purpose of this page
+
+Provide guidance on how to change from one IP pool to another on a running cluster.
+
+## Prerequisites
+
+**{{site.prodname}} IPAM**
+
+This guide only applies if you are using {{site.prodname}} IPAM.
+
+**Orchestrator support**
+
+While {{ site.prodname }} supports changing IP pools, not all orchestrators do.
+Be sure to consult the documentation for whichever installer you are using to ensure it supports changing it.
+
+For example, in Kubernetes, all three of the following arguments must be equal to, or contain, the Calico IP pool CIDRs:
+
+- kube-apiserver's `--pod-network-cidr`
+- kube-proxy's `--cluster-cidr`
+- kube-controller-manager's `--cluster-cidr`
```

Comment:
It is a little problematic to make possessives out of these components...a little too casual IMO. I would replace each "'s" with ":" (i.e., switch out the possessive for a colon)